tokio/src/sync/semaphore.rs

```diff
@@ -140,12 +132,12 @@ impl Semaphore {
     /// The semaphore must be wrapped in an [`Arc`] to call this method.
     ///
```

Comment:
again, maybe worth noting that an error is returned when the semaphore is closed?